src/theory/arith/arith_rewriter.cpp

```diff
@@ -897,6 +899,10 @@ RewriteResponse ArithRewriter::rewriteExtIntegerOp(TNode t)
 RewriteResponse ArithRewriter::postRewriteIAnd(TNode t)
 {
   Assert(t.getKind() == Kind::IAND);
+  if (!d_expertEnabled)
```

Comment:
rather than do this case by case, it would be better to create a category of "expert rewrites" and test in the first switch if you are in that category, then call the postRewriteExpert etc. Like you did for relations.

Reply:
Sure, I will update.